cms/djangoapps/api/v1/tests/test_serializers/test_course_runs.py

```diff
@@ -61,8 +67,45 @@ def test_data(self, expected_pacing_type, self_paced):
                 },
             ],
             'images': {
-                'card_image': request.build_absolute_uri(course_image_url(course)),
+                'card_image': self.request.build_absolute_uri(course_image_url(course)),
             },
             'pacing_type': expected_pacing_type,
         }
+
+    @ddt.data(
+        ('instructor_paced', False),
+        ('self_paced', True),
+    )
+    @ddt.unpack
+    def test_data_with_enrollment_dates(self, expected_pacing_type, self_paced):
+
+        enrollment_start = self.start - datetime.timedelta(days=7)
+        enrollment_end = self.end - datetime.timedelta(days=14)
+        course = self.get_course(self_paced, enrollment_start, enrollment_end)
```

Comment:
#NIT instead of repeating this function twice, you can use `ddt.data` to figure out how to pipe in the enrollment_start and enrollment_end. But if it's too much work, feel free to ignore